cookiecutter/tap-template/{{cookiecutter.tap_id}}/meltano.yml

```diff
@@ -1,6 +1,10 @@
 version: 1
 send_anonymous_usage_stats: true
 project_id: "{{cookiecutter.tap_id}}"
+environments:
```

Comment:
```suggestion
default_environment: dev
environments:
```

Reply:
@pnadolny13, @WillDaSilva - Was thinking on this more... since this is a `meltano.yml` file going into a tap or target's repo, should we just name a single environment called `test` or `devtest` (or maybe just `dev`).

I don't know if `staging` and `prod` would apply here. We could add something like `cicd` but that's probably more confusing/distracting than helpful.

```yml
default_environment: test
environments:
- name: test
```

Reply:
And whatever we decide, should we add the same for the target cookiecutter?